src/olympia/scanners/migrations/0042_remove_waffle_switch.py

```diff
@@ -0,0 +1,15 @@
+# Generated by Django 2.2.12 on 2021-02-01 16:11
+from django.db import migrations
+
+
+def remove_waffle_switch(apps, schema_editor):
+    Switch = apps.get_model('waffle', 'Switch')
+
+    Switch.objects.filter(name='enable-scanner-results-api').delete()
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [('scanners', '0032_auto_20200519_1438')]
```

Comment:
this needs to depend on the previous migration (i.e. 41) - that's why all the tests are failing in CI

Reply:
Got it. Thank you

Reply:
What about now?